File: my-folder/3753-maximum-difference-between-even-and-odd-frequency-i/solution.cpp
class Solution {
public:
    int maxDifference(string s){
        int maxi=0,mini=INT_MAX; vector<int>mp(26,0);
        for(auto it:s) mp[it-'a']++;
        for(auto it:mp) {if(it%2==1||it==0) maxi=max(maxi,it);else mini=min(mini,it);}
        return maxi-mini;
    }
};

File: my-folder/3753-maximum-difference-between-even-and-odd-frequency-i/solution.py
class Solution:
    def maxDifference(self,s):
        maxi = 0
        mini = float('inf')
        mp = [0] * 26
        for it in s:
            mp[ord(it) - ord('a')] += 1
        for it in mp:
            if it % 2 == 1 or it == 0:
                maxi = max(maxi, it)
            else:
                mini = min(mini, it)
        return maxi - mini

        
